raiden/utils/__init__.py

```diff
@@ -223,22 +222,6 @@ def channel_to_api_dict(channel_):
     }
 
 
-def channelstate_to_api_dict(channel_state):
```

Comment:
Humm, this PR was introducing this function ... it will be used by the rest api when the channel state is used

Reply:
I see. When looking at the code as of this PR I could not find a usage and it was the point of a cyclic dependency so I removed it. You can reintroduce it without the cyclic dependency in the next PR I suppose.